components/kubeflow/katib-launcher/src/launch_study_job.py

```diff
@@ -0,0 +1,161 @@
+                      help='Direction of optimization. minimize or maximize.')
+  parser.add_argument('--objectivevaluename', type=str,
+                      help='Objective value name which trainer optimizes.')
+  parser.add_argument('--optimizationgoal', type=float,
+                      help='Stop studying once objectivevaluename value ' +
+                           'exceeds optimizationgoal')
+  parser.add_argument('--requestcount', type=int,
+                      default=1,
+                      help='The times asking request to suggestion service.')
+  parser.add_argument('--metricsnames', type=strToList,
+                      help='StudyJob metrics name list.')
+  parser.add_argument('--parameterconfigs', type=yamlOrJsonStr,
+                      default={},
+                      help='StudyJob parameterconfigs.')
+  parser.add_argument('--nasConfig',type=yamlOrJsonStr,
+                      default={},
+                      help='StudyJob nasConfig.')
+  parser.add_argument('--workertemplatepath', type=str,
+                      default="",
+                      help='StudyJob worker spec.')
+  parser.add_argument('--mcollectortemplatepath', type=str,
+                      default="",
+                      help='StudyJob worker spec.')
+  parser.add_argument('--suggestionspec', type=yamlOrJsonStr,
+                      default={},
+                      help='StudyJob suggestion spec.')
+  parser.add_argument('--studyjobtimeoutminutes', type=int,
+                      default=10,
+                      help='Time in minutes to wait for the StudyJob to complete')
+  
+  args = parser.parse_args()
+
+  logging.getLogger().setLevel(logging.INFO)
+
+
+  logging.info('Generating studyjob template.')
+  template_file = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'hp.template.yaml')
+  content_yaml = _generate_studyjob_yaml(template_file, args.name, args.namespace, args.optimizationtype, args.objectivevaluename,
+                                         args.optimizationgoal, args.requestcount, args.metricsnames, args.parameterconfigs,
+                                         args.nasConfig, args.workertemplatepath, args.mcollectortemplatepath, args.suggestionspec)
+
+  config.load_incluster_config()
+  api_client = k8s_client.ApiClient()
+  create_response = study_job_client.create_study_job(api_client, content_yaml)
+  job_name = create_response['metadata']['name']
+  job_namespace = create_response['metadata']['namespace']
+
+  expected_condition = ["Completed", "Failed"]
+  wait_response = study_job_client.wait_for_condition(
+      api_client, job_namespace, job_name, expected_condition, 
+      timeout=datetime.timedelta(minutes=args.studyjobtimeoutminutes))
+  succ = False
+  if wait_response.get("status", {}).get("condition") == "Completed":
+    succ = True
+    trial = get_best_trial(wait_response["status"]["bestTrialId"])
+    with open('/output.txt', 'w') as f:
```

Comment:
Please get this path from the command line.

Reply:
done! thanks!